android/assets/jsons/translations/Dutch.properties

```diff
@@ -899,7 +895,7 @@ Unbuildable = Onbouwbaar
 # Promotions
 
 Pick promotion = Kies een promotie
- OR  = OF
+OR  = OF
```

Comment:
Spaties aan het begin of einde zijn onderdeel van de vertaling, dus als je ze zomaar weghaalt dan werkt de vertaling niet meer